01-Back-End Criação de Usuário/CustomersController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace _01_Back_End_Criação_de_Usuário
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CustomersController : ControllerBase
+    {
+
+        private readonly ICustomersRepository _customers;
+        public CustomersController(ICustomersRepository customers)
+        {
+            _customers = customers;
+        }
+
+        [HttpPost]
+        public virtual ActionResult Create([FromBody] CustomersModel model)
+        {
+            return StatusCode(_customers.Create(model));
+        }
+
+        [HttpGet("Get All Customers")]
```

Comment:
```suggestion
        [HttpGet]
```
Nos padrões de endpoint devemos deixar as palavras em minúsculo, sem caracteres especiais e se for necessário separar palavras, usar hífen.
Como a endpoint já se trata de customers e você busca por todos customers **sem filtro**, podemos deixar sem endpoint personalizado.